editor/src/components/canvas/controls/select-mode/controls-common.tsx

```diff
@@ -10,6 +10,13 @@ export interface CSSNumberWithRenderedValue {
   renderedValuePx: number
 }
 
+export const pureCSSNumberWithRenderedValue = (
```

Comment:
pure sounds odd to me, would unitless be better?

```suggestion
export const unitlessCSSNumberWithRenderedValue = (
```

Reply:
great idea, done!